ext/civicrm_admin_ui/ang/afsearchPremiumProducts.aff.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use CRM_CivicrmAdminUi_ExtensionUtil as E;
+
+return [
+  'type' => 'search',
+  'title' => E::ts('Premium Products'),
+  'placement' => [],
+  'icon' => 'fa-list-alt',
+  'server_route' => 'civicrm/admin/contribute/managePremiums',
+  'is_public' => FALSE,
+  'permission' => [
+    'access CiviCRM',
```

Comment:
We need to match this to whatever permission was being checked previously for this screen.
I find the menu xml files a bit too magical in the way they assign permissions to paths, but I *think* the way it works is that the standard url inherits its permissions from the next path in the hierarchy that has them declared, and that would be `civicrm/admin`. And that one checks `administer CiviCRM system + administer CiviCRM data + access CiviCRM`. That seems a bit silly. Maybe we should set this to
```suggestion
    'access CiviContribute',
    'administer CiviCRM data',
```